dotCMS/src/test/java/com/dotcms/filters/NormalizationFilterTest.java

```diff
@@ -86,15 +109,16 @@ public void test_uri_normalization_valid_URI() throws IOException, ServletExcept
         String originalURI = "/folder/important/secret_file.dat";
         compare(originalURI, originalURI);
 
+        // (query strings are not part of URI)
         originalURI = "/folder/folder1/forward_jsp.jsp?FORWARD_URL=http://google.com";
-        compare(originalURI, originalURI);
+        compare(originalURI, originalURI.substring(0,originalURI.indexOf("?")));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [String.indexOf(char) is faster than String.indexOf(String).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5436097)